prism.el

```diff
@@ -940,6 +950,11 @@ arguments to set the same faces."
                         (blend ag bg alpha)
                         (blend ab bb alpha)))))
 
+(defun prism-opacity (color percent)
```

Comment:
Since this just calls prism-blend, is it really necessary to define a new function?

Reply:
I really think so. Opacity is a common parameter, the sort of thing users customizing things are likely to understand and look for- I was at least, and I actually wrote my own blend here before realizing you already had one. It is an important, and not immediately obvious observation that opacity here is the blending of the prism color with their background. It also uses percents to match the color-lightens-name and color-desaturate-name format- but I should make the name match as well.

Reply:
Okay, but my question remains.  :)  What would be the benefit of adding this as a new function rather than just calling `prism-blend` where it's needed?